packages/EasyStandard/src/Rector/SingleLineCommentRector.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Rector;
+
+use Nette\Utils\Strings;
+use PhpParser\Comment;
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * @see \EonX\EasyStandard\Tests\Rector\SingleLineCommentRector\SingleLineCommentRectorTest
+ */
+final class SingleLineCommentRector extends AbstractRector
+{
+    /**
+     * @var string[]
+     */
+    public $disallowedEnd = ['...', '.', ',', '?', ':', '!'];
+
+    /**
+     * @var string[]
+     */
+    public $shouldBeStartFromLowerCase = [
```

Comment:
Let's name this as `$ignoredPatterns` and check a regex pattern instead of a simple string. This will allow us to skip any arbitrary comments.

Reply:
Reworked.

Reply:
Please use a regex, not only `startsWith`.

Reply:
Done.